specification/cosmos-db/resource-manager/Microsoft.DocumentDB/preview/2021-04-01-preview/cosmos-db.json

```diff
@@ -5243,14 +5243,28 @@
             "schema": {
               "$ref": "#/definitions/ServiceResource"
             },
-            "description": "The cluster configuration parameters."
+            "description": "The Service configuration parameters."
           }
         ],
         "responses": {
           "200": {
-            "description": "The service cluster status.",
-            "schema": {
-              "$ref": "#/definitions/ServiceResource"
+            "description": "The service status.",
```

Comment:
This would be ServiceResource, 
"schema": {
              "$ref": "#/definitions/ServiceResource"
              }
              
              
              In the definition of ServiceResource we will use a discriminator.